geometry/hold.js

```diff
@@ -1,6 +1,13 @@
 import { Group } from './Group.js';
-import { inItem } from './inItem.js';
-import { on } from './on.js';
 
-export const hold = (geometry, geometries) =>
-  on(geometry, inItem(geometry), (inside) => Group([inside, ...geometries]));
+export const hold = (geometry, geometries) => {
```

Comment:
What does hold do?

Reply:
Ah, it causes an item to include another piece of content.